node/ChainwebNode.hs

```diff
@@ -529,7 +529,7 @@ pkgInfoScopes =
 -- SERVICE DATE for version 2.19
 --
 serviceDate :: Maybe String
-serviceDate = Just "2023-09-07T00:00:00Z"
+serviceDate = Just "2023-11-07T00:00:00Z"
```

Comment:
These are different days of the week. IIRC we always used to do Thursday, 00:00:00 UTC. Do we want to break with with this tradition?

Reply:
Oof okay the release calculator will need a fix here.